public/app/features/browse-dashboards/types.ts

```diff
@@ -3,7 +3,7 @@ import { CellProps, Column, HeaderProps } from 'react-table';
 import { DashboardViewItem, DashboardViewItemKind } from 'app/features/search/types';
 
 export type DashboardTreeSelection = Record<DashboardViewItemKind, Record<string, boolean | undefined>> & {
-  $all: boolean;
+  $all?: boolean;
```

Comment:
why is this now optional? 🤔

Reply:
thanks for catching this. i... can't remember, i can't think of a reason why. i undid the change and everything still seems fine so 🤷🏻